pkg/lib/aws/servicequotas.go

```diff
@@ -200,111 +201,114 @@ func (c *Client) VerifyNetworkQuotas(
 		return errors.WithStack(err)
 	}
 
-	err = c.ServiceQuotas().ListServiceQuotasPages(
-		&servicequotas.ListServiceQuotasInput{
-			ServiceCode: aws.String("vpc"),
-		},
-		func(page *servicequotas.ListServiceQuotasOutput, lastPage bool) bool {
-			if page == nil {
-				return false
-			}
-			for _, quota := range page.Quotas {
-				if quota == nil || quota.QuotaCode == nil || quota.Value == nil {
-					continue
+	if !clusterAlreadyExists {
```

Comment:
Add a comment explaining why these checks can be skipped when running a configure.